### ISSUE TITLE
Wrong default arguments in new Python API

### ISSUE THREAD

**Issue by stefaniapedrazzi** [cyberbotics/webots#5569]
Some default argument specified in the documentation are not specified in the new Python API implementation.
For example this instruction ``object.enablePoseTracking(timestep)`` is throwing the following error:
```
TypeError: Node.enablePoseTracking() missing 1 required positional argument: 'fromNode'
```

These are some functions where the default argument is documented but not implemented:
* Supervisor.enablePoseTracking
* Supervisor.disablePoseTracking
* Supervisor.getPose
* Supervisor.worldSave

Not sure if these were skipped because it is difficult to pass a NULL pointer to the C functions.
But the documentation should match the implementation and if possible the default argument should be kept.

**Comment by stefaniapedrazzi** [comment]
Does it work correctly for you?

For the fix in #5572 doesn work.
Instead of correctly accepting the default or `None` argument, I get this error message in the console:
```
Error: wb_supervisor_node_enable_pose_tracking() called with a NULL or invalid 'node' argument.
```

This is wrong because `NULL` (or `None`) should be a valid argument.

**Comment by stefaniapedrazzi** [comment]
Strange.. after a `make clean`. It is now working!
